src/utils.rs

```diff
@@ -146,54 +148,84 @@ impl Console {
         disconnect_timeout_sec: Option<u64>,
     ) -> NitroCliResult<()> {
         // Initialize variables
-        let (epoll, mut events, mut epoll_timeout_us, mut start_epoll_time) = (
+        let (epoll, mut events) = (
             Epoll::new().map_err(|e| {
```

Comment:
I can't say I understand why these are declared in a tuple. Seems pretty weird, especially since that tuple is not used anywhere else; they should just be declared normally, one after the other.

Reply:
I have changed the variable declaration to no longer be a tuple.